update.go

```diff
@@ -151,8 +156,9 @@ other options are ignored.
 			for opt, dest := range map[string]*uint64{
 				"cpu-period":         r.CPU.Period,
 				"cpu-quota":          r.CPU.Quota,
-				"cpu-share":          r.CPU.Shares,
+				"cpu-shares":         r.CPU.Shares,
```

Comment:
The main intent of PR is to add the kernelTCP limits, along with that I had modified this flag too
Yes flag name for cpu-share changed. I have updated the update.bats with cpu.shares too.